editor/test/resources/test_project/graphics/box.model

```diff
@@ -1,4 +1,9 @@
-name: "box"
 mesh: "/graphics/box.dae"
-material: "/graphics/simple.material"
-textures: "/graphics/block.png"
+skeleton: ""
+animations: ""
+default_animation: ""
+name: "box"
+materials {
+  name: "default"
+  material: "/graphics/simple.material"
+}
```

Comment:
I think we need to check if the configuration you described existed in the file formats at some point. In that case I think we need to migrate it somehow.

And the fact that this file exists in the test content suggests that this was a valid model at some point in the past. 😬

Likely because we didn't match the textures by name at all in the past. I think they were just spread across the samplers in the shader in the order they were listed.

Reply:
Yes, restoring it does not affect the tests.